### ISSUE TITLE
[1.12.2] Bug - Moving Wand Removes Inventory - Mod Interaction (Refined Storage)

### ISSUE THREAD

**Issue by EpicGazel** [romelo333/notenoughwands#81]
#### Steps to reproduce:
1. Using the Moving Wand, right-click a crafter with an inventory
2. Using the Moving Wand, right-click a new space to replace Crafter
3. Check Crafter Inventory

#### Version:

- Minecraft: 1.12.2
- Forge: 14.23.4.2729
- Not Enough Wands: 1.12-1.7.0

**Comment by McJty** [comment]
This is probably something that is hard to fix as this has to do with how that crafter is implemented. Maybe this block should be blacklisted from the moving wand.

**Comment by xenoflot** [comment]
It's worse than just crafters! Using a moving wand to relocate the disk drive resulted in all the disks vanishing into thin air! Thank goodness for autobackups!

Current release of FTB Revelations
NEW 1.12-1.7.0
Forge 14.23.4.2759

**Comment by McJty** [comment]
yes, don't move refined storage stuff with the moving wand. These should be added to a blacklist

**Comment by corey8084** [comment]
Would it be possible to set it up that if you pick up one of those items, rather than deleting the inventory, it gets spit out (similar to when mining the disk drive)?
